Traducir.Web/src/App/Components/SuggestionsTable.tsx

```diff
@@ -86,6 +99,9 @@ export default class SuggestionsTable extends React.Component<SuggestionsTablePr
                 } else {
                     _that.props.showErrorMessage(null, e.response.status);
                 }
+                this.setState({
```

Comment:
you should use `_that` instead of this... when the promise is resolved, `this` may not refer to the component

Reply:
ok great let me review how react work this things.. and fixed that..

Reply:
you can see that I'm using `_that` on line 100 ;) it's just replacing `this` with `_that`